cli/lib/toCubeShape/Dimension.js

```diff
@@ -1,26 +1,31 @@
 const clownface = require('clownface')
 const rdf = require('rdf-ext')
-const TermMap = require('@rdfjs/term-map')
 const TermSet = require('@rdfjs/term-set')
 const ns = require('@tpluscode/rdf-ns-builders')
 
-const datatypeParsers = new TermMap([
-  [ns.xsd.date, term => new Date(term.value)],
-  [ns.xsd.double, term => parseFloat(term.value)],
-  [ns.xsd.float, term => parseFloat(term.value)],
-  [ns.xsd.integer, term => parseInt(term.value)],
-  [ns.xsd.decimal, term => parseFloat(term.value)],
-  [ns.xsd.gYear, term => parseInt(term.value)],
-])
+const datatypeParsers = (datatype) => {
+  switch (datatype.value) {
+    case ns.xsd.date.value:
+      return term => new Date(term.value)
+    case ns.xsd.double.value:
+    case ns.xsd.float.value:
+    case ns.xsd.decimal.value:
+      return term => parseFloat(term.value)
+    case ns.xsd.integer.value:
+    case ns.xsd.int.value:
+    case ns.xsd.gYear.value:
+      return term => parseInt(term.value)
+  }
+}
```

Comment:
I suppose implemented like this is a little easier on the eye and similar in performance. Could improve by memorising the individual cases or changing them into contents.

By the way, do we want to include other `xsd:integer` subtypes and `xsd:dateTime` here too? @bergos